app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -1,65 +1,68 @@
 <template>
   <div>
-    <v-row class="mt-6" no-gutters>
-      <!-- page title -->
-      <v-col cols="12" sm="6" order="2" order-sm="1">
-        <h1>{{ $t('trans.formDesigner.formDesign') }}</h1>
-      </v-col>
-      <!-- buttons -->
-      <v-col class="text-right" cols="12" sm="6" order="1" order-sm="2">
-        <v-tooltip bottom>
-          <template #activator="{ on, attrs }">
-            <v-btn
-              class="mx-1"
-              @click="onExportClick"
-              color="primary"
-              icon
-              v-bind="attrs"
-              v-on="on"
-            >
-              <v-icon>get_app</v-icon>
-            </v-btn>
-          </template>
-          <span>{{ $t('trans.formDesigner.exportDesign') }}</span>
-        </v-tooltip>
-        <v-tooltip bottom>
-          <template #activator="{ on, attrs }">
-            <v-btn
-              class="mx-1"
-              @click="$refs.uploader.click()"
-              color="primary"
-              icon
-              v-bind="attrs"
-              v-on="on"
-            >
-              <v-icon>publish</v-icon>
-              <input
-                class="d-none"
-                @change="loadFile"
-                ref="uploader"
-                type="file"
-                accept=".json"
-              />
-            </v-btn>
-          </template>
-          <span>{{ $t('trans.formDesigner.importDesign') }}</span>
-        </v-tooltip>
-      </v-col>
-      <!-- form name -->
-      <v-col cols="12" order="3">
-        <h3 v-if="name">{{ name }}</h3>
-      </v-col>
-      <!-- version number-->
-      <v-col cols="12" order="4">
-        <em
-          >{{ $t('trans.formDesigner.version') }} :
-          {{ this.displayVersion }}</em
-        >
-      </v-col>
-    </v-row>
-    <BaseInfoCard class="my-6">
+    <div :class="{ 'dir-rtl': isRTL }">
+      <v-row class="mt-6" no-gutters>
+        <!-- page title -->
+        <v-col cols="11">
```

Comment:
Is there a reason to switch to using 11 columns for the title? Or should it be 12, like before?